bindings/python/pymongoarrow/api.py

```diff
@@ -11,10 +11,25 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+import functools
 import warnings
 
 from pymongoarrow.context import PyMongoArrowContext
 from pymongoarrow.lib import process_bson_stream
+from pymongoarrow.schema import Schema
+
+
+__all__ = [
+    'aggregate_arrow_all',
+    'find_arrow_all',
+    'Schema'
+]
+
+
+_PATCH_METHODS = [
```

Comment:
I listed these as strings instead of function handles because the names are not yet declared at this point in the file. I didn't particularly like the idea of declaring `_PATCH_METHODS` at the bottom of the file so I did this.

Reply:
Cool, I think either way is fine.